lib/AthomApi.js

```diff
@@ -98,11 +99,11 @@ class AthomApi {
           name: 'code',
```

Comment:
```suggestion
          name: 'receivedCode',
```

needs to match the name you are destructuring from the result